src/main/resources/subscriptionsTemplate.json

```diff
@@ -42,5 +46,8 @@
                 }
             ]
         }
-    ]
+
+    ],
+    "subscriptionName" : "Subscription_Test_Modify",
```

Comment:
Remove subscriptionName line here. You have already one subscriptionName line in the beginning:  "subscriptionName" : "Subscription2" ....